### PR TITLE
Add to description: this is Nextcloud server

### PR DESCRIPTION
**Pull request by mabkenar**
Mention in the package description that this is the Nextcloud server part, not the client. This is currently not mentioned anywhere, and is a bit confusing.

### CODE REVIEW

**Comment by kyrofa** [comment]
The summary doesn't help?

**Comment by mabkenar** [comment]
Oh I see! I just saw the snap page in [uApp Explorer](https://uappexplorer.com/app/nextcloud.nextcloud) which only shows "Nextcloud" in the summary. Maybe it is lagging behind this repo?

Anyway, having a little more description of the package never hurts :)

File: snapcraft.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: nextcloud
 version: 11.0.1snap3
 summary: Nextcloud Server
 description: |
- Access, share and protect your files, calendars, contacts, communication and
+ Nextcloud server running on Apache with MySQL. Access, share, and protect your files, calendars, contacts, communications, and
  more at home and in your enterprise.
```
Comment (kyrofa) [inline snapcraft.yaml]: Sorry for the delay on this. Would it work to say "Access, share and protect your files, calendars, contacts, communication and more at home and in your enterprise with the Nextcloud Server." ?

**Comment by kyrofa** [comment]
I just realized this PR is targeting master. Please read the [contribution guide](https://github.com/nextcloud/nextcloud-snap/blob/master/CONTRIBUTING.md).

**Pull request closed without merging**